docs/reference/setup/sysconfig/configuring.asciidoc

```diff
@@ -50,7 +52,35 @@ the `limits.conf` file:
 elasticsearch  -  nofile  65536
 --------------------------------
 
-This change will only take effect the next time the `elasticsearch` user opens
+We recommend setting <<bootstrap.memory_lock,`bootstrap.memory_lock`>>.
+In order for Elasticsearch to be able to lock the heap in memory through 
+mlockall and avoid swapping, add the following line to the `limits.conf` 
```

Comment:
These days the recommended approach would be to create a file in `/etc/security/limits.d/` (e.g., `elasticsearch.conf`). I think that while we are here, we should go the extra mile and change these recommendations to be centered around this.